src/_includes/layouts/home.njk

```diff
@@ -44,6 +44,7 @@
 	<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.js"></script>
 	<script src="https://www.canada.ca/etc/designs/canada/wet-boew/js/wet-boew.min.js"></script>
 	<script src="https://www.canada.ca/etc/designs/canada/wet-boew/js/theme.min.js"></script>
+  {% include "partials/sa11y.njk" %}
```

Comment:
Adding the if statement to the layout instead of in the partial and adding it to the "local" environment too

```suggestion
{%- if settings.env == "dev" or settings.env == "local" -%}
    {% include "partials/sa11y.njk" %}
{%- endif -%}
```